metpy/calc/tests/test_thermo.py

```diff
@@ -416,3 +417,24 @@ def test_most_unstable_parcel():
     assert_almost_equal(ret[0], 959.0 * units.hPa, 6)
     assert_almost_equal(ret[1], 22.2 * units.degC, 6)
     assert_almost_equal(ret[2], 19.0 * units.degC, 6)
+
+
+def test_sbcape():
+    """Tests the surface-based CAPE and CIN calculation."""
+    p = np.array([959., 779.2, 751.3, 724.3, 700., 269.]) * units.mbar
+    temperature = np.array([22.2, 14.6, 12., 9.4, 7., -38.]) * units.celsius
+    dewpoint = np.array([19., -11.2, -10.8, -10.4, -10., -53.2]) * units.celsius
+    cape, cin = sbcape_sbcin(p, temperature, dewpoint)
+    assert_almost_equal(cape, 58.0368212 * units('joule / kilogram'), 6)
+    assert_almost_equal(cin, -89.8073512 * units('joule / kilogram'), 6)
+
+
+def test_sbcape_profile():
+    """Tests the basic CAPE and CIN calculation."""
+    p = np.array([959., 779.2, 751.3, 724.3, 700., 269.]) * units.mbar
+    temperature = np.array([22.2, 14.6, 12., 9.4, 7., -38.]) * units.celsius
+    dewpoint = np.array([19., -11.2, -10.8, -10.4, -10., -53.2]) * units.celsius
+    parcel_prof = parcel_profile(p, temperature[0], dewpoint[0])
```

Comment:
Assuming we keep the profile kwarg and this test still exists, we should not be calling the `parcel_profile` function here, but including an array of values. This helps reduce the coupling of the tests which will be good as we start refactoring how profiles work.